packages/fds-components/src/style/interactive/buttonGroup.css

```diff
@@ -10,62 +16,99 @@
   margin-left: -1px;
   position: relative;
   border-radius: 0;
+  z-index: 2;
+}
+
+/* z-index is used here to manage the border for those button
+states and their stacking order (whether they display or not).
+The border for a disabled button should appear below a border
+for a button being clicked on (:active) */
+.btngroup .groupbtn:disabled {
+  z-index: 1;
+}
+
+.btngroup .groupbtn.active {
```

Comment:
This selector triggers some lint errors. Worth discussing? I can describe why I'm doing all of this. If it does make sense, what's the command we're using to ignore lint errors?

Reply:
What if instead of `.groupbtn.active` we create a `.groupbtn--active` class?

Reply:
_great_ suggestion. Made that change.